addons/sys_gui/XEH_postInit.sqf

```diff
@@ -14,3 +14,28 @@ acre_player addEventHandler ["Take", {call FUNC(handleTake)}];
 [15, [false, false, false]], true] call CBA_fnc_addKeybind;
 
 ["unit", FUNC(onVolumeControlKeyPressUp)] call CBA_fnc_addPlayerEventHandler;
+
+DFUNC(enterVehicle) = {
+    params ["_player"];
+    disableSerialization;
+
+    private _ctrl = ctrlParentControlsGroup (uiNamespace getVariable ["ACRE_VEH_INFO", controlNull]);
+
+    if (!isNull objectParent _player) then {
+        _ctrl ctrlShow true;
+    } else {
+        _ctrl ctrlShow false;
+    };
+};
+
+// Handle showing the display at the very begining when unit is on foot
+[{
+    params ["_player"];
+    [_player] call FUNC(enterVehicle);
+}, [acre_player], 0.1] call CBA_fnc_waitAndExecute;
+
+// Add event handler
+["vehicle", {
+    params ["_player"];
+    [_player] call FUNC(enterVehicle);
+}] call CBA_fnc_addPlayerEventHandler;
```

Comment:
Can't you use the apply retro for player EH?
https://github.com/CBATeam/CBA_A3/blob/master/addons/events/fnc_addPlayerEventHandler.sqf#L22

Reply:
Thanks for the tip! I totally forgot about that.